app/assets/javascripts/services/faye_messages.jsx

```diff
@@ -0,0 +1,52 @@
+function message_box(name) {
+  
+    $(function() {
+    var client = new Faye.Client('http://localhost:9292/faye');
+    var time = new Date();
+    var message_to_bottom = document.getElementById('list');
+
+    $('#new_message_form').submit(function(){
+      if (matches = $('#message').val().match(/@(\w+) (.+)/)) {
+        client.publish('/messages/private/' + matches[1], {
+          username: name,
+          msg: matches[2],
+          time: time.toLocaleTimeString()
+        });
+        if(matches[1] !== name){
+          var form_time = time.toLocaleTimeString();
```

Comment:
if we have more than 4 var in one place use it through coma separator for example `
            var create_data = document.createElement('p'),
               paragraph_with_text = form_time +' - '+name+': '+matches[2],
               document.createTextNode(paragraph_with_text),
               paragraph = create_data.appendChild(add_time);`